stable/artifactory-ha/values.yaml

```diff
@@ -1570,3 +1570,366 @@ filebeat:
+        <labels>
+          host ${hostname}
+        </labels>
+      </source>
+
+      # input plugin that collects metrics for in_tail plugin
+      <source>
+        @type prometheus_tail_monitor
+        <labels>
+          host ${hostname}
+        </labels>
+      </source>
+
+      <source>
+        @type tail
+        @id access_service_tail
+        path "#{ENV['JFROG_HOME']}/log/access-service.log"
+        pos_file "#{ENV['JFROG_HOME']}/log/access-service.log.pos"
+        tag jfrog.rt.access.service
+        <parse>
+            @type multiline
+            format_firstline /\d{4}-\d{1,2}-\d{1,2}/
+            format1 /^(?<timestamp>[^ ]*) \[(?<service_type>[^\]]*)\] \[(?<log_level>[^\]]*)\] \[(?<trace_id>[^\]]*)\] \[(?<class_line_number>.*)\] \[(?<thread>.*)\] - (?<message>.+)$/
+            time_key timestamp
+            time_format %Y-%m-%dT%H:%M:%S.%LZ
+        </parse>
+      </source>
+      <source>
+        @type tail
+        @id artifactory_service_tail
+        path "#{ENV['JFROG_HOME']}/log/artifactory-service.log"
+        pos_file "#{ENV['JFROG_HOME']}/log/artifactory-service.log.pos"
+        tag jfrog.rt.artifactory.service
+        <parse>
+            @type multiline
+            format_firstline /\d{4}-\d{1,2}-\d{1,2}/
+            format1 /^(?<timestamp>[^ ]*) \[(?<service_type>[^\]]*)\] \[(?<log_level>[^\]]*)\] \[(?<trace_id>[^\]]*)\] \[(?<class_line_number>.*)\] \[(?<thread>.*)\] - (?<message>.+)$/
+            time_key timestamp
+            time_format %Y-%m-%dT%H:%M:%S.%LZ
+        </parse>
+      </source>
+      <source>
+        @type tail
+        @id frontend_service_tail
+        path "#{ENV['JFROG_HOME']}/log/frontend-service.log"
+        pos_file "#{ENV['JFROG_HOME']}/log/frontend-service.log.pos"
+        tag jfrog.rt.frontend.service
+        <parse>
+            @type multiline
+            format_firstline /\d{4}-\d{1,2}-\d{1,2}/
+            format1 /^(?<timestamp>[^ ]*) \[(?<service_type>[^\]]*)\] \[(?<log_level>[^\]]*)\] \[(?<trace_id>[^\]]*)\] \[(?<class_line_number>.*)\] \[(?<thread>.*)\] - (?<message>.+)$/
+            time_key timestamp
+            time_format %Y-%m-%dT%H:%M:%S.%LZ
+        </parse>
+      </source>
+      <source>
+        @type tail
+        @id metadata_service_tail
+        path "#{ENV['JFROG_HOME']}/log/metadata-service.log"
+        pos_file "#{ENV['JFROG_HOME']}/log/metadata-service.log.pos"
+        tag jfrog.rt.metadata.service
+        <parse>
+            @type multiline
+            format_firstline /\d{4}-\d{1,2}-\d{1,2}/
+            format1 /^(?<timestamp>[^ ]*) \[(?<service_type>[^\]]*)\] \[(?<log_level>[^\]]*)\] \[(?<trace_id>[^\]]*)\] \[(?<class_line_number>.*)\] \[(?<thread>.*)\] - (?<message>.+)$/
+            time_key timestamp
+            time_format %Y-%m-%dT%H:%M:%S.%LZ
+        </parse>
+      </source>
+      <source>
+        @type tail
+        @id router_service_tail
+        path "#{ENV['JFROG_HOME']}/log/router-service.log"
```

Comment:
Wouldn't it make more sense to set this environment variable to something like:
```
JFROG_LOGS_DIR={{ .Values.artifactory.persistence.mountPath }}/log
```
?
This would save the extra `/log`

Reply:
definitely a better idea, I'll push up the change